tests/scripts/run-throughput-perf.py

```diff
@@ -290,6 +295,11 @@ def runIterations(dll_name, dll_path, iterations, crossgen_path, jit_path, assem
     if opt_level == 'min_opt':
         my_env['COMPlus_JITMinOpts'] = '1'
 
+    if jit_name == 'legacy':
+        my_env['COMPLUS_AltJit'] = '*'
```

Comment:
Could you please specify what is "right"?
I have tested and results were pretty different for "COMPlus_AltJitName=", "COMPlus_AltJitName=legacyjit.dll" and "COMPlus_AltJitName=legacyjit.dll && COMPlus_JITMinOpts=1" so I think it is the correct set of EnvVars.